src/main/java/org/terasology/thirst/event/DrinkConsumedEvent.java

```diff
@@ -17,5 +17,6 @@
 
 import org.terasology.entitySystem.event.Event;
 
+/** This event is sent to an entity to indicate that it will be drinking a drink. */
```

Comment:
~"will be drinking"~ "has drunk" - the event name is in past tense, indicating that the drink was already consumed.